### PR TITLE
ASIMOV: human being --> "person of a NT approved crew species"

### PR DESCRIPTION
**Pull request by silicons**
guess people don't like reading silicon policy ah well let's get rid of the contradiction

### CODE REVIEW

File: code/game/objects/items/AI_modules.dm [hunk 1]
```diff
@@ -362,7 +362,7 @@ AI MODULES
 /obj/item/aiModule/core/full/asimov
 	name = "'Asimov' Core AI Module"
 	law_id = "asimov"
-	var/subject = "human being"
+	var/subject = "person of an NT approved crew species"		//CITADEL CHANGED FROM HUMANS!
```
Comment (Ghommie) [inline code/game/objects/items/AI_modules.dm:365]: Pretty verbose, this equivales to `NT approved crew species` (or `specimen`) and still as vague either way without explicit reference to the concept of roundstart races.
Also you may want to text macro `[subject]s` to `[subject]\s` at line 373.

**Comment by Armyguy** [comment]
Is this not what the Crewsimov law set is for?

**Comment by Poojawa** [comment]
literally just AI ban people who are being stupid about this.

**Comment by deathride58** [comment]
@Armyguy Crewsimov is for protecting the station's crew and only the station's crew, meaning lings, off-station antags, ERT, centcom officers, etc, are all unprotected by crewsimov. Asimov is for protecting humans in general, and nobody else, which leads to some fun shit like wizards and nukeops sharing the same exact silicon protection as the crew.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**